### PR TITLE
Modified filtering for inventory table

### PR DESCRIPTION
**Pull request by katelyn-schmidt**
solves issue #56 
inventory table filtering ignores case and checks for substring (instead of just start of name)

### CODE REVIEW

File: src/components/DataTable.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ class DataTable extends React.Component {
                   Header: "Item",
                   accessor: row => `${row.Item}`,
                   filterMethod: (filter, row) => {
-                    return row._original.Item.startsWith(filter.value);
+                    return row._original.Item.toLowerCase().includes(
+                      filter.value
```
Comment (trev4ev) [inline src/components/DataTable.js]: can you make the filter value also lowercase so that it still matches if they type with uppercase

**Review by trev4ev — CHANGES_REQUESTED** [review]
1 small change

**Review by trev4ev — APPROVED** [review]
lgtm!

**Comment by katelyn-schmidt** [comment]
made requested change, please check it out! @trev4ev

**Review by abaruah117 — APPROVED** [review]
looks great :)

**Pull request merged**